csharp/native/DynamicSubmission/Worker/DynamicSubmission.cs

```diff
@@ -0,0 +1,454 @@
+                                             taskOptions);
+
+          logger_.LogDebug("Submitting Aggregation");
+
+          //var FinalResultId = taskHandler.ExpectedResults.Single();
+
+          var payload = await taskHandler.CreateResultsAsync(new List<CreateResultsRequest.Types.ResultCreate>
+                                                             {
+                                                               new()
+                                                               {
+                                                                 Data = UnsafeByteOperations.UnsafeWrap(Encoding.ASCII.GetBytes(JsonSerializer
+                                                                                                                                  .Serialize(tableInput))),
+                                                                 Name = "Payload",
+                                                               },
+                                                             });
+
+          var payloadId = payload.Results.Single()
+                                 .ResultId;
+
+          await taskHandler.SubmitTasksAsync(new List<SubmitTasksRequest.Types.TaskCreation>
+                                             {
+                                               new()
+                                               {
+                                                 PayloadId = payloadId,
+                                                 ExpectedOutputKeys =
+                                                 {
+                                                   resultId,
+                                                 },
+                                                 DataDependencies =
+                                                 {
+                                                   subTaskResultIds,
+                                                 },
+                                               },
+                                             },
+                                             taskOptions);
+        }
+        else
+        {
+          logger_.LogDebug("Debug : Enter in subtask process");
+
+          uint result = 0;
+
+          for (uint i = 0; taskHandler.DataDependencies.Count == 0 && i < size ; i++)
+          {
+            result += tableInput.Values[i];
+          }
+          // if we have dependencies this mean that it is the aggregation task
+          // We add all result of the dependencies together to get the final result
+          if (taskHandler.DataDependencies.Count > 0)
+          {
+            logger_.LogDebug("Debug : Enter in aggregation process");
+
+            var results = taskHandler.DataDependencies.Values.Select(res => uint.Parse(Encoding.ASCII.GetString(res)));
+
+            for (var i = 0; i < taskHandler.DataDependencies.Count; i++)
+            {
+              logger_.LogInformation("Info : result = {result}\n", result);
```

Comment:
```suggestion
              logger_.LogInformation("Info : result = {result}", result);
```
This should be avoided even more